api/workAllocation/constants/actions.ts

```diff
@@ -1,60 +1,60 @@
 const ASSIGN = {
-  id: 'assign',
```

Comment:
What's happened to all the indenting in here? It's off.

Reply:
File shouldn't be in there anyway as discussed, had removed but hadn't pushed yet